network/codec/roundTripHeader_test.go

```diff
@@ -0,0 +1,40 @@
+package codec_test
+
+// pushd network ; BINSTAT_LEN_WHAT="~net=99;~lock=99;~Backend=99" BINSTAT_ENABLE=1 GO111MODULE=on go test -v -coverprofile=coverage.txt -covermode=atomic --tags relic ./codec/roundTripHeader_test.go ; popd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/onflow/flow-go/model/messages"
+	"github.com/onflow/flow-go/network"
+	cborcodec "github.com/onflow/flow-go/network/codec/cbor"
+	jsoncodec "github.com/onflow/flow-go/network/codec/json"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+func roundTripHeaderViaCodec(t *testing.T, codec network.Codec) {
```

Comment:
Please add godoc describing the test scenario.

Reply:
@yhassanzadeh13 thanks for the comment! Added godoc as requested! :-)